src/screens/Students.js

```diff
@@ -149,25 +163,23 @@ class Students extends Component {
   renderLoadingSubroute() {
     return (
       <SubRouteWrapper>
-        <SpinnerOverlay open/>
+        <SpinnerOverlay open />
       </SubRouteWrapper>
     );
   }
 
-  renderStudentListItem = student => (
-    <StudentListItem
-      key={student.id}
-      student={student}
-      onClick={this.handleStudentClick}
-    >
-      <StudentEditButton onClick={this.handleStudentEditClick.bind(null, student)}>
-        <EditIcon/>
-      </StudentEditButton>
-    </StudentListItem>
-  );
+  @action.bound renderStudentListItem(student) {
```

Comment:
This isn't a "side-effecty" function so it doesn't need the `@action.bound`. Refer to "When to use actions?": https://mobx.js.org/refguide/action.html.

Reply:
Granted, I've seen it used like this. The one benefit is access to `this` without having to think about it, but using an arrow function allows `this` to be bound as you'd want it to.